ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activityNumericAnswer.component.ts

```diff
@@ -47,35 +58,10 @@ export class ActivityNumericAnswer implements OnInit, OnChanges {
         }
     }
 
-    public onBlur(value: string): void {
-        const answer = parseInt(value, 10);
-        if (isNaN(answer)) {
-            this.numericField.setValue('');
-            this.emitAnswer(null);
-        } else if (this.numericField.invalid) {
-            this.emitAnswer(null);
-        } else {
-            this.numericField.setValue(answer);
-            this.emitAnswer(answer);
-        }
-    }
-
-    private emitAnswer(value: number | null): void {
-        this.block.answer = value;
-        this.valueChanged.emit(value);
-    }
-
     private initForm(): void {
-        const numericValidators = [];
-        if (_.isNumber(this.block.min)) {
-            numericValidators.push(Validators.min(this.block.min));
-        }
-        if (_.isNumber(this.block.max)) {
-            numericValidators.push(Validators.max(this.block.max));
-        }
         this.numericField = new FormControl({
             value: _.isNumber(this.block.answer) ? this.block.answer : '',
             disabled: this.readonly
-        }, numericValidators);
+        }, {updateOn: 'blur'});
     }
```

Comment:
Mostly removed stuff. Only thing of note is that the control emits on 'blur' (the default is 'onchange'). This was what original was doing, so going with that.